client/src/store/store.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright © 2022-2024, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { subscribeWithSelector } from "zustand/middleware";
+import { StateCreator, createStore } from "zustand/vanilla";
+
+import { AppStoreState, initialState } from "./initialState";
+import { createLogActions } from "./log/actions";
+import { LogActions } from "./log/actions";
+import { logger } from "./middleware";
+import { createRunActions } from "./run/actions";
+import { RunActions } from "./run/actions";
+
+export type AppActions = LogActions & RunActions;
+export type AppStore = AppActions & AppStoreState;
+
+const store: StateCreator<AppStore, []> = (...params) => ({
+  ...initialState,
+
+  ...createLogActions(...params),
+  ...createRunActions(...params),
+});
+
+export const useStore = createStore<AppStore>()(
+  subscribeWithSelector(logger(store)),
```

Comment:
Having a bunch of `console.info` statements in our log doesn't seem ideal for "production" environments. How often do we dump things to the console? Should this be something that is limited to development?

Reply:
fixed in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/b53a6ba752b9c595d468a795ee1a79100fc84c3c.